src/views/partials/labels.handlebars

```diff
@@ -0,0 +1,18 @@
+<h3>Labels</h3>
+{{!-- List Labels --}}
+<form>
+    {{#each label}}
+        <input type="checkbox" checked id="card_label_{{@index}}">
+        <label for="card_label_{{@index}}"><p class="card-label-highlight" style="background-color: {{this.color}};"></p></label>
```

Comment:
the labels should be the color, but also the label text next to it. I would just add another <p> tag after this p tag and do like {{this.text}} inside it

Reply:
Added additional p tag for label text